packages/web-app-files/src/components/FilesList/ResourceTable.vue

```diff
@@ -648,6 +663,16 @@ export default {
 </script>
 <style lang="scss">
 .resource-table {
+  &-resource-wrapper {
+    &:hover > .resource-table-edit-name {
+      opacity: 1;
+    }
+  }
+  &-edit-name {
+    display: inline-flex;
+    vertical-align: super;
```

Comment:
Not sure if this is necessary since we're using flexbox?

Reply:
tried to do it without it and didn't center vertically, no idea why the other div for the resource is centered fine :/